orchestrator-salt/src/main/resources/salt/salt/top.sls

```diff
@@ -53,6 +53,9 @@ base:
     - prometheus.server
     - ambari.server
     - jdbc.connectors
+{% if not salt['file.directory_exists']('/yarn-private') %}  # FIXME (BUG-92637): must be disabled for YCloud
```

Comment:
This should be moved to server_start to support gateway recovery.

Reply:
Moved it.